vms/platformvm/state/state.go

```diff
@@ -2364,21 +2369,21 @@ func (s *state) writeSubnetOwners() error {
 	return nil
 }
 
-func (s *state) writeSubnetManagers() error {
-	for subnetID, manager := range s.subnetManagers {
+func (s *state) writeSubnetConversions() error {
+	for subnetID, c := range s.subnetConversions {
 		subnetID := subnetID
-		manager := manager
-		delete(s.subnetManagers, subnetID)
+		c := c
```

Comment:
Can/should we remove this local copying within the for loop closure now that we're using Golang 1.22?

Reply:
yes good call